core/io/zip_io.h

```diff
@@ -39,6 +39,13 @@
 #include "thirdparty/minizip/unzip.h"
 #include "thirdparty/minizip/zip.h"
 
+// get the current file info and safely convert the full filepath to a String
+int godot_unzip_get_current_file_info(unzFile p_zip_file, unz_file_info64 &r_file_info, String &r_filepath);
+// try to locate the file in the archive specified by the filepath (works with large paths and Unicode)
+int godot_unzip_locate_file(unzFile p_zip_file, String p_filepath, bool p_case_sensitive = true);
+
+//
```

Comment:
Stray comment.
Also the comments seem to be bit long and describing what can be mostly understood from function name and parameters. Maybe we can shorten it. Just my opinion.

Reply:
Huh, stray comment?

I admit that the comments I added are edging on verbose... but they were all placed with reason. In this case I'd rather document why this function is different (and preferred) over `unzLocateFile` because it's not immediately obvious why this function is needed when minizip provides another.

If the code has bounds errors or safety problems I'd rather put more work into the comments. If you want to reword anything, have at it!

Reply:
In this case it's internal APIs so not suitable for documenting in the XML docs. Seems fine as is to me.

Reply:
Cool.

@Macksaur When I said Stray comment I meant those empty double slashes.

Reply:
Oh, right! haha! I was delimiting the top part of the file (the exposed minizip support) from the rest of the file that implements zip I/O.